Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.yml

```
@@ -0,0 +1,245 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: CrowdStrike Indicator Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- display: CrowdStrike Base URL
+  name: base_url
+  required: true
+  type: 0
+  defaultvalue: "https://api.crowdstrike.com/"
+- display: CrowdStrike API Client ID
+  displaypassword: CrowdStrike API Client Secret
+  additionalinfo: 'For non 6.1 users username is CrowdStrike API Client ID, password is CrowdStrike API Client Secret.'
```

In 5.5 and above the username field will be displayed as `CrowdStrike API Client ID` so no need to specify it.
```suggestion
  additionalinfo: 'For non 6.1 users - enter your  CrowdStrike API Client Secret in the password field.
```